package.json

```diff
@@ -23,14 +22,5 @@
   },
   "eslintConfig": {
     "extends": "react-app"
-  },
-  "browserslist": [
```

Comment:
why removed these

Reply:
dependency problem, has to be removed.
More description can refer to this [issue](https://github.com/wangwenyue/wangwenyue.github.io/issues/1)

Reply:
This problem has been solved, more detail refer to this [issue](https://github.com/wangwenyue/wangwenyue.github.io/issues/2)

Reply:
> dependency problem, has to be removed.
> More description can refer to this [issue](https://github.com/wangwenyue/wangwenyue.github.io/issues/1)

This problem has been solved, more detail refer to this [issue](https://github.com/wangwenyue/wangwenyue.github.io/issues/2)